game/subsystems/art/bullet.py

```diff
@@ -0,0 +1,27 @@
+import pygame,math
+from pygame.locals import *
+class Bullet(pygame.sprite.Sprite):
+    def __init__(self,imageName):
+        self.image = pygame.transform.scale(pygame.image.load("%s.jpg" %imageName), (40,40))
```

Comment:
keep in mind images probably won't be in the art directory; we'll want to put images in their own folder